.travis.yml

```diff
@@ -122,7 +122,7 @@ jobs:
       compiler: gcc
       env: ARCH=x86_64
       before_install:
-        - docker run -w /src -dit --name alpine -v $(pwd):/src node:13-alpine
```

Comment:
Isn't this worse since it leaves us vulnerable to CI breakages when LTS changes.

Also, isn't 13 the current LTS making this change a no-op? (https://nodejs.org/en/about/releases/)

Reply:
LTS is 12 according to the download site: https://nodejs.org/en/download/ (and this PR does pass).

It's true we are vulnerable to CI breakage when the LTS changes, but at that point we really should do something - if not even the LTS can run our code, then our code is wrong. https://i.kym-cdn.com/photos/images/original/000/645/713/888.jpg